### ISSUE TITLE
Cannot apply style_class on date filter

### ISSUE THREAD

**Issue by darron1217** [vedmack/yadcf#469]
`style_class` not working for date filter (datepicker)

Current
![image](https://user-images.githubusercontent.com/8064923/41029690-4b64c16a-69b7-11e8-8761-587127ccb564.png)

Expected
![image](https://user-images.githubusercontent.com/8064923/41029733-6f118bca-69b7-11e8-98cb-8ba10d92ad99.png)

The code
```
                  column_number : 4,
                  filter_type: 'date',
                  filter_default_label: '검색',
                  style_class: 'form-control',
                  reset_button_style_class: 'form-control',
                  filter_plugin_options: {
                    language: 'ko',
                    startView: "months",
                    minViewMode: "months"
                  }
```

On version 0.9.3

**Comment by vedmack** [comment]
please provide a minimal test page jsfiddle...

**Comment by darron1217** [comment]
http://jsfiddle.net/chcLmmps/78/

It's quite dirty but you can see the bug

**Comment by lenamtl** [comment]
I think you have to edit the JS directly.
This is what I did to apply Bootstrap CSS on the fields.

**Comment by vedmack** [comment]
I already fixed that and will publish it very soon

**Comment by vedmack** [comment]
fixed in 0.9.4.beta.1
